sui-execution/suivm/sui-adapter/src/execution_engine.rs

```diff
@@ -522,8 +522,10 @@ mod checked {
                     pt,
                 )
             }
-            TransactionKind::AuthenticatorStateUpdate(_auth_state_update) => {
-                todo!();
+            TransactionKind::AuthenticatorStateCreate(_)
+            | TransactionKind::AuthenticatorStateUpdate(_)
+            | TransactionKind::AuthenticatorStateExpire(_) => {
+                panic!("AuthenticatorState{{Create|Update|Expire}} should not exist in suivm");
```

Comment:
Heads up @dariorussi, when you merge `suivm` back into `latest`, you shouldn't merge this part.

Also, I'm guessing that (a) this change is going to land before your cut of `v1` and therefore you'll need to redo the cut.